src/backfill-many.py

```diff
@@ -0,0 +1,52 @@
+# backills many articles using a simple text file of paths to XML.
+# paths may include globs.
+# all paths are verified to exist before execution.
+# see `/backfill-many.sh`
+
+import glob, os, sys
+import adhoc_backfill
+
+def main(xml_repo_dir, backfill_file, dry_run):
```

Comment:
`backfill_file` argument is perhaps not used yet in this function, maybe goes in place of "backfill.txt" on line 13?

Reply:
good catch